packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.ts

```diff
@@ -20,6 +20,11 @@ import { FileUri } from '@theia/core/lib/node/file-uri';
 import URI from '@theia/core/lib/common/uri';
 import { inject, injectable } from '@theia/core/shared/inversify';
 import { SearchInWorkspaceServer, SearchInWorkspaceOptions, SearchInWorkspaceResult, SearchInWorkspaceClient, LinePreview } from '../common/search-in-workspace-interface';
+import * as fs from '@theia/core/shared/fs-extra';
+import path = require('path');
```

Comment:
Can likely be converted:

```suggestion
import * as path from 'path';
```